src/main/java/com/arpnetworking/metrics/mad/Aggregator.java

```diff
@@ -122,6 +125,33 @@ public void notify(final Observable observable, final Object event) {
 
         final Record record = (Record) event;
         final Key key = new DefaultKey(record.getDimensions());
+
+        _periodicMetrics.recordGauge(
+                "aggregator/samples",
+                record.getMetrics().values().stream()
+                        .map(m -> {
+                            final ImmutableList<CalculatedValue<?>> countStatistic = m.getStatistics().get(
+                                    STATISTIC_FACTORY.getStatistic("count"));
+                            if (countStatistic != null) {
+                                return countStatistic.stream()
+                                        .map(s -> s.getValue().getValue())
+                                        .reduce(Double::sum)
+                                        .orElse(0.0d)
+                                        +
+                                        m.getValues().size();
```

Comment:
Just to sanity check - you're summing both the count (if available) and the number of values because the protocol allows for both aggregate statistics and raw samples, correct?

Reply:
Yes, that's correct.